aptos-move/e2e-move-tests/src/harness.rs

```diff
@@ -68,11 +66,16 @@ impl MoveHarness {
         let acc = Account::new_genesis_account(addr);
         // APTOS has 8 decimals so this is only 1000 coins.
         let data = AccountData::with_account(acc, 100_000_000_000, 10);
-        self.txn_seq_no.insert(addr, 10);
         self.executor.add_account_data(&data);
+        self.txn_seq_no.insert(addr, 10);
         data.account().clone()
     }
 
+    /// Gets the account for aptos root.
+    pub fn aptos_root_account(&mut self) -> Account {
+        self.new_account_at(AccountAddress::ONE)
+    }
```

Comment:
The Aptos root account used to be `0xA5501C`. has this changed?  There's already a function for that one

Reply:
But the framework is still deployed to 0x1. I need an account for 0x1 here because I upgrade framework code.

Reply:
Okay, let's call it `aptos_framework_account` to prevent confusion.